packaging/THIRD-PARTY.txt

```diff
@@ -0,0 +1,111 @@
+
+Lists of 109 third-party dependencies.
+     (Apache License, Version 2.0) Titanium JSON-LD 1.1 (JRE11) (com.apicatalog:titanium-json-ld:1.3.1 - https://github.com/filip26/titanium-json-ld)
+     (Apache License, Version 2.0) jcommander (com.beust:jcommander:1.82 - https://jcommander.org)
```

Comment:
@thadguidry Thanks for working on this! Is this PR to a stage where we can remove this `THIRD-PARTY.txt` file from git? I think it would be good to keep it dynamically generated if it's not too expensive, given how quickly our dependencies evolve, no?

Reply:
That was the intent, to generate dynamically for that reason.  I'd love the help to finish things off and merge it with the necessary changes that you see fit.  Does it all just work ok for you if you do `mvn package` as a test locally?  Because I still get issues, but then I get timeouts quite often from open sources mirrors and have to use the ones in China, so maybe it's a local issue.  The timeout changes I did helped but not fully for me.  As I did this before in the past in America, things just worked, but that was for a much smaller dependency tree than what we have.

Be my guest!